sycl/test-e2e/Matrix/SG32/get_coordinate_ops.cpp

```diff
@@ -10,7 +10,7 @@
 // REQUIRES: aspect-ext_intel_matrix
 // REQUIRES-INTEL-DRIVER: lin: 30049, win: 101.4943
 
-// XFAIL: !igc-dev
+// XFAIL: !igc-dev || arch-intel_gpu_pvc
 // XFAIL-TRACKER: GSD-6376
```

Comment:
xfail tracker should be different for !igc-dev and arch-intel_gpu_pvc, as there are different root causes for fails
I think you can make 2 XFAIL lines instead of ||

Reply:
Thanks so much for the suggestions! I have update the test.